app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/HomeFragment.java

```diff
@@ -55,6 +60,31 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         checkUserCredentials();
     }
 
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+
+        if(getContext()!=null){
+            if(Utils.isNetworkConnected(getContext())){
+                Call<Search> call =  apiClient.getTotalProductCount();
+                call.enqueue(new Callback<Search>() {
+                    @Override
+                    public void onResponse(Call<Search> call, Response<Search> response) {
+                        textHome.setText("Open Food Facts is an open database of more than "+response.body().getCount()+" food products (plus the ones you'll contribute) with ingredients, allergens, nutrition facts and all the tidbits of information we can find on product labels.");
```

Comment:
Use string resource to display the results. This is not translatable.

Reply:
Missed that one

Reply:
@huzaifaiftikhar Hardcoded String.